playbook-website/app/javascript/site_styles/_opacity_guidelines.scss

```diff
@@ -0,0 +1,48 @@
+ul {
+   
+  &.opacity-play {
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: none;
+  background: none;
+  border-radius: none;
+  border: 0px;
+  padding: none;
+  flex-wrap: wrap;
+  }
+}
+
+.selector { 
```

Comment:
Lets improve this class name? What is this? is this a container of content? perhaps it is containing the opacity guidelines. Why not name it `.opacity-guideline-container {`